images-list

```diff
@@ -130,6 +132,10 @@ ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operat
 ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operator 3.17.1
 ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operator 3.17.3
 ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operator 3.17.4
+ghcr.io/epinio/epinio-server rancher/epinio-server 0.7.1
+ghcr.io/epinio/epinio-server rancher/mirrored-epinio-server 0.7.1
+ghcr.io/epinio/epinio-ui rancher/epinio-ui v0.6.2-0.0.1
+ghcr.io/epinio/epinio-ui rancher/mirrored-epinio-ui v0.6.2-0.0.1
```

Comment:
This should be rancher/mirrored-epinio-epinio-ui

Reply:
Will fix.